recipes/conda-project/meta.yaml

```diff
@@ -18,11 +18,11 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.7
```

Comment:
Is there a reason to support Python 3.7? We had originally planned not to, since it will be end-of-life in ~2 months. But if there is some `conda-forge` requirement, then we may have to reconsider that decision.

Reply:
Oh, nevermind. Looks like our source actually support 3.7. Thanks for the fix here.